hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/RecordListBuilder.java

```diff
@@ -249,6 +259,9 @@ public void revertChildrenOf(@NonNull final SingleTransactionRecordBuilderImpl r
         if (childRecordBuilders == null) {
             childRecordBuilders = new ArrayList<>();
         }
+        if (precedingTxnRecordBuilders == null) {
+            precedingTxnRecordBuilders = new ArrayList<>();
+        }
```

Comment:
Seems we could instantiate `precedingTxnRecordBuilders` as a list on object creation, but this is fine

Reply:
Agreed